geniza/corpus/models.py

```diff
@@ -409,7 +409,13 @@ class Document(ModelIndexable):
     )
 
     footnotes = GenericRelation(Footnote, related_query_name="document")
-    log_entries = GenericRelation(LogEntry, related_query_name="document")
+
+    @property
+    def log_entries(self):
+        return LogEntry.objects.filter(
+            object_id=self.id,
+            content_type_id=ContentType.objects.get_for_model(Document),
```

Comment:
Could we get content type id once somewhere so we don't have to load it every time?

Or here at least we could filter on the content object name and app name, instead of looking up the id every time.

Reply:
LogEntry actually doesn't have a `content_object` field, it only stores the object ID and content type ID, so I'll do the former!